roles/oneview_label/README.md

```diff
@@ -0,0 +1,33 @@
+## oneview_label
+Manage OneView Label resources.
+
+#### Synopsis
+Provides an interface to manage Label resources. Can create, update, or delete.
+
+#### Requirements (on the host that executes the module)
+  * python >= 2.7.9
+  * hpeOneView >= 5.4.0
```

Comment:
labels resource is refactored in python 6.1.0. So please mention the same

Reply:
done